issue #6062 resolved  - Successful message in account detail & addresses

HoseinHabibiyan opened the pull request:
PR to close #6062

RomanovM reviewed (changes requested):
@HoseinHabibiyan  There are unwanted changes in [defaultResources.nopres.xml](https://github.com/nopSolutions/nopCommerce/pull/6231/files#diff-a1af1655b1f2658635d918ca78a9acede9b4654549240968a8a3fe8b556e340b) (with line endings I suppose), so all this file was overwritten. Could you fix this, please? 
Also, a new locale should be added to [LocalizationMigration](https://github.com/nopSolutions/nopCommerce/blob/develop/src/Presentation/Nop.Web.Framework/Migrations/UpgradeTo460/LocalizationMigration.cs).

HoseinHabibiyan commented:
> @HoseinHabibiyan There are unwanted changes in [defaultResources.nopres.xml](https://github.com/nopSolutions/nopCommerce/pull/6231/files#diff-a1af1655b1f2658635d918ca78a9acede9b4654549240968a8a3fe8b556e340b) (with line endings I suppose), so all this file was overwritten. Could you fix this, please? Also, a new locale should be added to [LocalizationMigration](https://github.com/nopSolutions/nopCommerce/blob/develop/src/Presentation/Nop.Web.Framework/Migrations/UpgradeTo460/LocalizationMigration.cs).

yes sure . fixed that.
added locale to LocalizationMigration .

RomanovM commented:
@HoseinHabibiyan Thanks for your help.
